app/components/Article/article.css

```
@@ -130,3 +130,12 @@ article .footer-comtainer > * {
 h1.teal {
   color: var(--colors-teal-500);
 }
+@media only screen and (max-width: 640px) {
+  article .footer-comtainer {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  article .footer-comtainer > div:nth-child(-n + 2) {
```

Is it a good practice to use `div:nth-child` as used? Perhaps an additional class would be clearer when reading the code and less likely to break unexpectedly? I don't have a strong feeling here, just thought I would ask.
```suggestion
  article .footer-comtainer .footer-section {
```

My CSS knowledge is limited, so my apologies if the rationale for the current approach is obvious 😄

---

Both options are fine by me. @mruwnik do you prefer nth-child or a new class?